src/lib/frontend/d_cnf.ml

```diff
@@ -1817,7 +1844,7 @@ let make dloc_file acc stmt =
     | { id = DStd.Id.{name = Simple name; _}; contents = `Hyp t; loc; attrs;
         implicit=_ } ->
       let name =
-        match DStd.Tag.get t.term_tags DE.Tags.named with
+        match DStd.Tag.get t.term_tags lemma_attr with
```

Comment:
Do we want to fall back on `:named` if it is provided but `:lemma` is not provided?